lib/project_types/script/layers/domain/extension_point.rb

```diff
@@ -42,14 +44,21 @@ def initialize(config)
           end
 
           def all
-            @all ||= @config.map do |language, libray_config|
-              ExtensionPointLibrary.new(language, libray_config)
+            @all ||= @config.merge(wasm_config).map do |language, library_config|
+              ExtensionPointLibrary.new(language, library_config)
             end
           end
 
           def for(language)
             all.find { |ep| ep.language == language }
           end
+
+          private
+
+          def wasm_config
+            return {} unless ShopifyCLI::Feature.enabled?(:scripts_beta_languages)
+            { "wasm" => { "repo" => DEFAULT_REPO } }
+          end
```

Comment:
At this point, what if we just move the wasm config into [extension_points.yml](https://github.com/Shopify/shopify-cli/blob/edd5ce86fa6b64e6b01f002d636ccf133663a134/lib/project_types/script/config/extension_points.yml)? Also relates to your comment [here](https://github.com/Shopify/shopify-cli/pull/1905/files#r789011704).

Reply:
Hmmm, the goal of this was so that api-developers would not need to include wasm in that file... BUT this was before we decided on using sparse-checkout for wasm as well.
Since the wasm create depends on the sparse-checkout repo being setup correctly, I think it's important to provide it manually now. 

Good catch! 👍